src/utils/setup/create-community-champion-profile-pages.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+import { communityChampions } from '../../queries/community-champions';
+
+const createCommunityChampionProfilePage = async (champion, createPage) => {
+    const { firstName, lastName } = champion;
+    const slug = `/community-champions/${firstName.toLowerCase()}-${lastName.toLowerCase()}`;
```

Comment:
Do any of the champions have any non-letter characters such as spaces or apostraphes in their names? For example d'arte has an `'` and would need to be escaped or such for a URL?

Reply:
thanks for bringing this up! yes there is the ö in Michael Höller. should i encode them by using `encodeURIComponent`?

Reply:
yes!

Reply:
this also got me thinking...what if we have two champions with the same name? i know this is extremely unlikely but would having the slug be something that's guaranteed to be unique like /community-champions/<champion_id> be better?

Reply:
it's failing to create Michael's page when I use `encodeURIComponent`...seems like a known issue - https://github.com/gatsbyjs/gatsby/issues/16765 🤔

Reply:
Would https://github.com/gatsbyjs/gatsby/issues/16765#issuecomment-632400169 help?

Two with the same name seems a bit improbable tbh so not sure we need to worry about that. Also design/product has asked for the URL to contain the name for ease-of-sharing purposes

Reply:
> Would [gatsbyjs/gatsby#16765 (comment)](https://github.com/gatsbyjs/gatsby/issues/16765#issuecomment-632400169) help?

page does get created if i use `decodeURIComponent` although im not sure what the point of decoding the uri component is if it was not encoded before? it just behaves the same as before without the encoding/decoding with the slug being /community-champions/michael-höller

> Two with the same name seems a bit improbable tbh so not sure we need to worry about that. Also design/product has asked for the URL to contain the name for ease-of-sharing purposes

ok! :)

Reply:
yeah hmm I'm also a bit confused here. If we don't need to decode then let's just not decode then my bad